sn_cli/src/subcommands/wallet.rs

```diff
@@ -90,6 +97,18 @@ pub enum WalletCmds {
         #[clap(name = "transfer")]
         transfer: String,
     },
+    /// Listen for transfer notifications from the network over gossipsub protocol,
```

Comment:
Just a little suggestion here. You could change this doc comment to:
```
    /// Listen for transfer notifications over gossipsub protocol.
    /// 
    /// Transfers will be deposited to a local wallet.
    ///
    /// Only cash notes owned by the public key corresponding to the provided SK will be accepted,
    /// verified to be valid against the network, and deposited onto a locally stored wallet.
```

It currently looks like this on the `wallet` subcommand help:
```
Commands for wallet management

Usage: safe wallet [OPTIONS] <COMMAND>

Commands:
  address         Print the wallet address
  balance         Print the wallet balance
  deposit         DEPRECATED will be removed in future versions.
                      Prefer using the send and receive commands instead.
  create          Create a local wallet from the given (hex-encoded) Secret Key
  get-faucet      Get tokens from a faucet
  send            Send a transfer
  receive         Receive a transfer created by the 'send' command
  receive-online  Listen for transfer notifications from the network over gossipsub protocol, depositing them onto a local wallet. Only cash notes owned by the public key corresponding to the provided SK will be accepted, verified to be valid against the network, and deposited onto a locally stored wallet
  help            Print this message or the help of the given subcommand(s)
```

However, if you use a header/summary format, it will change it to this:
```
Commands for wallet management

Usage: safe wallet [OPTIONS] <COMMAND>

Commands:
  address         Print the wallet address
  balance         Print the wallet balance
  deposit         DEPRECATED will be removed in future versions.
                      Prefer using the send and receive commands instead.
  create          Create a local wallet from the given (hex-encoded) Secret Key
  get-faucet      Get tokens from a faucet
  send            Send a transfer
  receive         Receive a transfer created by the 'send' command
  receive-online  Listen for transfer notifications over gossipsub protocol
  help            Print this message or the help of the given subcommand(s)
```

Then when you use `--help` on the `receive-online` command, you get the full documentation.

Just looks a little bit tidier in my opinion.